### DISCUSSION TITLE
Add notification for updates

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#11013] [Dashboard]
Added this as Issue, but seems more appropriate as FR so please consider adding that to the frontend: 

Frontend no longer displays a notification, or sends a persistent notification about updates. (might not have done all of this before, so not a bug , but a question)

### Describe the behavior you expected

please add a notification for any updates available I the configuration menu:

<img width="1060" alt="Schermafbeelding 2021-12-23 om 13 41 00" src="https://user-images.githubusercontent.com/33354141/147241910-d062c0e6-dcc2-4ecf-8aed-21b9f857cb05.png">

on the configuration menu item:

<img width="276" alt="Schermafbeelding 2021-12-23 om 13 41 23" src="https://user-images.githubusercontent.com/33354141/147241957-ef3c5df7-bbfd-40f2-8e69-98d1bc1336a4.png">

### What version of Home Assistant Core has the issue?

2022.2.0.dev20211222

### Additional information

We now have to explicitly check Configuration, while a notification would make it standout. And it would nicely align with the notifications we see in the Notification menu item, so no new interface aspect or learning curve

**Comment by spacegaier** [comment]
Persistent notifications are created by the backend (the frontend merely renders them).

**Reply by Mariusthvdb** [reply]
ofc, I see that. You're saying this has nothing to do with the Frontend? Somehow these notifications should be displayed on that Configuration menu item, and I wouldn't know the command available for that.

Second to that, not even sure if we should be talking persistent notifications? Maybe some other type of internal alert for updates is  available. I mean, they are shown in the panel above, so all backend info is there.
